tests/oracle/test-oracle.cpp

```diff
@@ -1407,6 +1407,62 @@ TEST_CASE("Bulk iterators", "[oracle][bulkiters]")
     sql << "drop table t";
 }
 
+struct function_creator_clob : function_creator_base
+{
+    function_creator_clob(soci::session & sql)
+        : function_creator_base(sql)
+    {
+        sql <<
+             "create or replace function soci_test(input in varchar2) return clob as "
+             "begin return input; end;";
+    }
+};
+
+TEST_CASE ( "Oracle clob", "[oracle][clob]" )
+{
+    soci::session sql ( backEnd, connectString );
+    std::string testPhrase{ "SystГЁme est ouvert aux requГЄtes d'information" };
```

Comment:
Is this intentionally misencoded?

Reply:
yes

Reply:
Well, first of all, this doesn't really work even with this PR apparently due to this.

Second, any I really don't want to be mean, but just I don't understand how can a so obviously questionable line be added without any comment explaining it. You should know that something so clearly unusual and apparently wrong would raise questions from whoever is reading it, and I can't fathom why you wouldn't want to explain that the string must be wrongly encoded to trigger the bug (if this is how it really is).